recipes-bsp/imx-mkimage/imx-boot_1.0.bb

```diff
@@ -58,20 +58,20 @@ IMXBOOT_TARGETS ?= \
                                                   'flash_multi_cores flash_dcd', d), d)}"
 
 BOOT_STAGING       = "${S}/${IMX_BOOT_SOC_TARGET}"
-BOOT_STAGING_mx8m  = "${S}/iMX8M"
-BOOT_STAGING_mx8dx = "${S}/iMX8QX"
+BOOT_STAGING:mx8m  = "${S}/iMX8M"
+BOOT_STAGING:mx8dx = "${S}/iMX8QX"
 
 SOC_FAMILY      = "INVALID"
-SOC_FAMILY_mx8  = "mx8"
-SOC_FAMILY_mx8m = "mx8m"
-SOC_FAMILY_mx8x = "mx8x"
+SOC_FAMILY:mx8  = "mx8"
+SOC_FAMILY:mx8m = "mx8m"
+SOC_FAMILY:mx8x = "mx8x"
 
 REV_OPTION ?= ""
-REV_OPTION_mx8qxp = \
+REV_OPTION:mx8qxp = \
     "${@bb.utils.contains('MACHINE_FEATURES', 'soc-revb0', '', \
                                                            'REV=C0', d)}"
 
-compile_mx8m() {
+compile:mx8m() {
```

Comment:
This (and consecutive substitutions) should not be made, as those functions are called by `do_compile()` and `do_deploy()`.

Changing the name here would lead to failures in executing `do_compile` and `do_deploy`.

I would suggest this substitution to be reverted for those internal functions.

Reply:
Good catch. @kraj can you do a build for an i.MX 8 and 8M machine, perhaps `imx8qm-mek` and `imx8mp-lpddr4-evk`? Maybe just a parse is all that's needed to test for this sort of problem?

Reply:
@thochstein I actually just did it for `imx8mp-lpddr4-evk`, and parsing was only breaking down in `imx-boot` recipe.

-- andrey

Reply:
@thochstein if I have time and resources I would do it but I dont sadly, so everyone has to do their bit, there are many machines in this layer, I cant test them all.